src/logius/fix-md-elements.js

```diff
@@ -0,0 +1,56 @@
+// @ts-check
+// Module logius/fix-table
+// add table class simple to all tables
+
+import { showWarning } from "../core/utils.js";
+export const name = "logius/fix-md-elements";
+
+export function run(conf) {
+  addClassTables(conf);
+  checkImgAlt(conf);
+  addClassCode(conf);
+}
+// todo check if algorithm is correct!
+function addClassTables(conf) {
+  // todo do nothing if this config is not set?
+  if (!conf.nl_markdownTableClass) {
+    conf.nl_markdownTableClass = "simple";
+  }
+  [...document.querySelectorAll("[data-format=markdown]:not(body)")].forEach(
+    section =>
+      section.querySelectorAll("table").forEach(table => {
+        // table.classList.add("complex");
+        table.classList.add(conf.nl_markdownTableClass);
+      })
+  );
+}
+
+function checkImgAlt(conf) {
```

Comment:
For this, you might want to consider creating a simple linter rule. For example:

https://github.com/w3c/respec/blob/develop/src/core/linter-rules/no-headingless-sections.js

Reply:
Thanks for the tip. A linter rule has been added to core and the "Occurred at" message has been localised.